features/step_definitions/stepdefs.js

```diff
@@ -894,7 +895,7 @@ Then(
 );
 
 Then(
-    'the object {string} should not be in the {string} collection',
+    'the object {string} should not be in the {string} collection', // Test failing for accept-follows.feature
```

Comment:
These two tests are failing after removing the kv-store based code. My assumption was that these should still work end to end after removing the old code.